package.json

```diff
@@ -52,8 +52,7 @@
   "dependencies": {
     "ajv": "^6.12.3",
     "bluebird": "^3.7.2",
-    "docker-progress": "^4.0.1",
-    "docker-toolbelt": "^3.3.8",
```

Comment:
`docker-toolbelt` declares its own dependency on dockerode v2.5.8, but I wanted to bump `dockerode` to v3 in the balena CLI -- and actually make use of dockerode v3 through `balena-multibuild`, rather than just have multiple copies of different versions of `dockerode` under the `node_modules` folder. I wanted to do this as part of the investigation on Node.js v14 support -- issue balena-io/balena-cli/issues/2165 -- and in any case, I expect that sooner or later we will have to update to dockerode v3. Updating `docker-toolbelt` would be an option, but it is in need of modernization (some legacy code can be deleted, coffeescript -> typescript conversion, bump dependencies) while I realized that the only reason to import it here and in `balena-cli` was to make use of the `getRegistryAndName` function, a synchronous regex utility function that does not have any dependencies of its own. So I copied that small-ish function to `balena-multibuild`, exported it for use by `balena-cli`, and removed the dependency on `docker-toolbelt` altogether.

I have only partially achieved the goal of using dockerode v3 through dependency injection in the CLI and `balena-multibuild`: both `docker-progress` and `balena-multibuild` now use dependency injection for the `dockerode` instance, so that the CLI can inject `dockerode` v3. However, `balena-multibuild` still declares a dependency on `dockerode` v2.5.8 for compatibility with [resin-docker-build](https://github.com/balena-io-modules/resin-docker-build). The next step would be to update `resin-docker-build`, but I need to work on other higher priority tasks and cannot continue this investigation right now. So I decided to merge the improvements I have made so far (this PR and a few others).